addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -477,9 +478,9 @@ def _import_fattura_pa(self, tree, invoice):
         if partner:
             invoice.partner_id = partner
         else:
-            message_to_log.append("%s<br/>%s" % (
+            message_to_log.append(Markup("%s<br/>%s" % (
```

Comment:
I don't know how to reproduce the problem, but this file was related to the need for markup in multiple sentences.
Using this PR (https://github.com/odoo/odoo/pull/122194) and this file (l10n_it_edi/models/account_move.py) the issue does not occur.
So, wherever markup was used in this file, I have changed it in the saas-16.3 file.
But, if not necessary, I remove it

Thanks :blush:

Reply:
The issue exists but I meant inside `_compose_info_message`, this should be markup
`output_str = Markup()` `output_str += Markup("</ul>")` `output_str += "<li>%s: %s</li>" % (line.tag, text)`
In addition to the current changes to be actually correct. Although I guess in general, if it parsed as xml, it's not a problem. Still good to be specific now to avoid problems in the future.